test/libethereum/blockchain.cpp

```diff
@@ -807,4 +807,19 @@ BOOST_AUTO_TEST_CASE(userDefinedFile)
 	dev::test::userDefinedTest(dev::test::doBlockchainTests);
 }
 
+BOOST_AUTO_TEST_CASE(rndCode)
```

Comment:
why did you place this test in this file?
This belongs in the FuzzTesting folder. For example in the helper filer.